<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>José Manuel Díez Botella</author>
 <class>buscasubcuenta_input</class>
 <widget class="QDialog" name="buscasubcuenta_input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar coeficientes de imputación</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel1">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>SUBCUENTA:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SubcuentaLineEdit">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>32767</height>
          </size>
         </property>
         <property name="maxLength">
          <number>40</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>TÍTULO:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TituloLineEdit">
         <property name="maxLength">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Selección</string>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QRadioButton" name="ComienceRadioButton">
          <property name="text">
           <string>&amp;Que comience por ...</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ContengaRadioButton">
          <property name="text">
           <string>Que contenga ...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Cuenta de Referencia:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="modelolineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="borramodelopushButton">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="copiarpushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>&amp;Copiar CIs</string>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/copiar.png</normaloff>:/iconos/graf/copiar.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CIBoton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Helvetica [Adobe]</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>&amp;Asignar CIs</string>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/edit.png</normaloff>:/iconos/graf/edit.png</iconset>
           </property>
           <property name="shortcut">
            <string>Alt+A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SalirBoton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Helvetica [Adobe]</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&amp;Salir</string>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
           </property>
           <property name="shortcut">
            <string>Alt+S</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="ctastableWidget">
     <column>
      <property name="text">
       <string>Código</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Descripción</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Imputaciones</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <includes>
  <include location="local">funciones.h</include>
 </includes>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>SalirBoton</sender>
   <signal>clicked()</signal>
   <receiver>buscasubcuenta_input</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
